### PR TITLE
To `polkadot-0.9.16`

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Review by xiaoch05 — APPROVED** [review]

File: node/runtime/pangoro/src/pallets/bridge_messages.rs [hunk 1]
```diff
@@ -17,8 +17,6 @@ frame_support::parameter_types! {
 		bp_pangolin::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
 	pub const MaxUnrewardedRelayerEntriesAtInboundLane: MessageNonce =
 		bp_pangolin::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
-	// 0x726f6f7400000000000000000000000000000000, b"root"
-	pub RootAccountForPayments: Option<AccountId> = Some(ConcatConverter::<_>::derive_substrate_address(&H160([0x72, 0x6f, 0x6f, 0x74, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])));
```
Comment (AsceticBear) [inline node/runtime/pangoro/src/pallets/bridge_messages.rs:21]: @xiaoch05 @HackFisher Please confirm that we can delete this safely.

Comment (HackFisher) [reply node/runtime/pangoro/src/pallets/bridge_messages.rs:21]: This is still being used, please do not delete it.

File: node/runtime/pangolin/src/pallets/bridge_messages.rs [hunk 2]
```diff
@@ -67,7 +63,7 @@ impl Config<WithPangolinParachainMessages> for Runtime {
 	type MaxUnrewardedRelayerEntriesAtInboundLane =
 		PangolinParachainMaxUnrewardedRelayerEntriesAtInboundLane;
 	type MessageDeliveryAndDispatchPayment =
-		FeeMarketPayment<Self, WithPangolinParachainFeeMarket, Ring, RootAccountForPayments>;
```
Comment (HackFisher) [inline node/runtime/pangolin/src/pallets/bridge_messages.rs:70]: Different versions of fee market?

File: node/runtime/pangoro/src/pallets/bridge_messages.rs (hunk 1 above)
Comment (xiaoch05) [reply node/runtime/pangoro/src/pallets/bridge_messages.rs:21]: how to pay the s2s relay fee for SourceRoot accont in new version?
This is used by remote governance

Comment (AurevoirXavier) [reply node/runtime/pangoro/src/pallets/bridge_messages.rs:21]: > how to pay the s2s relay fee for SourceRoot accont in new version? This is used by remote governance

Need to check from the upstream code. They deleted this.

File: node/runtime/pangolin/src/pallets/bridge_messages.rs (hunk 2 above)
Comment (AurevoirXavier) [reply node/runtime/pangolin/src/pallets/bridge_messages.rs:70]: @AsceticBear

File: node/runtime/pangoro/src/pallets/bridge_messages.rs (hunk 1 above)
Comment (AsceticBear) [reply node/runtime/pangoro/src/pallets/bridge_messages.rs:21]: I will raise a PR to resolve this.

File: node/runtime/pangolin/src/pallets/bridge_messages.rs (hunk 2 above)
Comment (AsceticBear) [reply node/runtime/pangolin/src/pallets/bridge_messages.rs:70]: Yes. In order to be consistence with the upstream repo, the newly fee market payment changed from [RootAccount](https://github.com/darwinia-network/darwinia-messages-substrate/blob/darwinia-v0.12.4/modules/fee-market/src/s2s/payment.rs#L56-L63)to  [submiter.linked_account](https://github.com/darwinia-network/darwinia-messages-substrate/blob/darwinia-v0.12.5/modules/fee-market/src/s2s/payment.rs#L60-L74) in this version.

File: node/runtime/pangoro/src/pallets/bridge_messages.rs (hunk 1 above)
Comment (AsceticBear) [reply node/runtime/pangoro/src/pallets/bridge_messages.rs:21]: https://github.com/darwinia-network/darwinia-common/pull/1452

File: node/runtime/pangolin/src/pallets/bridge_messages.rs (hunk 2 above)
Comment (AsceticBear) [reply node/runtime/pangolin/src/pallets/bridge_messages.rs:70]: https://github.com/darwinia-network/darwinia-common/pull/1452

**Comment by AsceticBear** [comment]
Please solve the conflicts.

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**